nexus/auth/src/authz/api_resources.rs

```diff
@@ -471,6 +471,67 @@ impl AuthorizedResource for IpPoolList {
     }
 }
 
+/// Synthetic, fleet-scoped resource representing the `/v1/multicast-groups`
+/// collection. This is not a persisted entity; it exists only to authorize
+/// collection-level actions on multicast groups.
+///
+/// Authorization derives from the parent Fleet (via the `parent_fleet`
+/// relation). Fleet Admins may create groups; Fleet Viewers may list them.
```

Comment:
I wonder if we really want to require fleet admin to create groups? Fleet is the right logical place in the hierarchy for a group to live. But we may want to allow regular users to create groups without being a fleet admin.

Reply:
I'll allow silo user to create/update groups.